src/Framework/QueryBuilder/Concerns/GroupByStatement.php

```diff
@@ -28,7 +28,7 @@ public function groupBy($tableColumn)
     protected function getGroupBySQL()
     {
         return ! empty($this->groupByColumns)
-            ? ['GROUP BY ' . implode(',', $this->groupByColumns)]
+            ? ['GROUP BY ' . implode(',', array_unique($this->groupByColumns))]
```

Comment:
Should we be compensating for redundant columns here? Or preventing them in the `$this->groupBy()` method? I'm thinking the latter.

Reply:
Yeah, I think you're right.

Reply:
Resolved: 854fd497